src/visual/Slider.js

```diff
@@ -932,9 +932,10 @@ export class Slider extends util.mix(VisualStim).with(ColorMixin, WindowMixin)
 
 		// marker:
 		this._marker = new PIXI.Graphics();
-		if (Number.isFinite(this._startValue))
+		const markerVal = Number.isFinite(this._rating) ? this._rating : (Number.isFinite(this._startValue) ? this._startValue : undefined);
```

Comment:
Please no nested ternary operator :-P
Other than that, I believe we are good to go.